cloud/vsphere/machineactuator.go

```diff
@@ -761,3 +771,18 @@ func pathExists(path string) (bool, error) {
 	}
 	return true, err
 }
+
+func createStartupScriptPath(dir, filename string) (string, error) {
+	tempDir, err := ioutil.TempDir("/tmp", "")
+	if err != nil {
+		return "", errors.New(fmt.Sprintf("Could not create temp dir for startup script. %v", err))
+	}
+	return path.Join(tempDir, filename), nil
+}
+
+func cleanUpStartupScript(fullPath string) {
+	glog.Infof("cleaning up startup script: %v", fullPath)
```

Comment:
Also log the machine name for which this startup script is being cleaned up since every machine's script is saved in the same path

Reply:
done.